src/components/DatePicker/DatePickInput/DatePickInput.tsx

```diff
@@ -6,28 +6,46 @@ import { flex } from '../../../theme/functions';
 import Icon from '../../Icon';
 import TextField from '../../TextField';
 import { wrapperStyle } from '../../TextField/TextField.style';
-import { DateRange } from '../DatePicker';
+import { DateFormatType, DateRange } from '../DatePicker';
 import { formFieldStyles } from '../../../theme/palette';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import localeData from 'dayjs/plugin/localeData';
+import { getLocaleFormat } from '../../../utils/helpers';
+import 'dayjs/locale/en-gb';
+import 'dayjs/locale/en';
 
 type Props = {
   isRangePicker: boolean;
   selectedDay: DateRange;
   inputLabel: string;
   /** Style of input field */
   styleType: formFieldStyles;
+  dateFormatOverride?: DateFormatType;
 } & DayPickerInputProps;
 
+(function InitDatePickerLocaleFormat(usLocale: string, euLocale: string) {
```

Comment:
this will define the whole app format in a bigger environment so i suggest to move it to the themeProvider level. What do you say?

Reply:
I agree. We can even create a new ts file (`initLocale.ts`) and import it to the index.tsx of the app.

Reply:
totally agree 💯